generator/src/routes/styles.css

```diff
@@ -1,9 +1,4 @@
-@import "@fontsource/fira-mono";
-
 :root {
```

Comment:
Are you actually using any of these variables or were they all from the SvelteKit example?

Reply:
I removed and unused css